fastify.js

```diff
@@ -262,6 +268,9 @@ function fastify (options) {
     },
     serializerCompiler: {
       get: function () { return this[kSerializerCompiler] }
+    },
+    version: {
+      get: function () { return this[kVersion] }
```

Comment:
Can we make the syntax consistent here at the same time? See line 256 above. So make all of these getters look like:

```suggestion
      get () { return this[kVersion] }
```